### PR TITLE
Read AWS credentials from ~/.aws/credentials even when ~/.bodylabs is present

### PR DESCRIPTION
**Pull request by paulmelnikow**
In this implementation, keys in ~/.bodylabs are still honored, but take priority over ~/.aws/credentials.

Without this change, we can't remove credentials from ~/.bodylabs.

### CODE REVIEW

File: baiji/config.py [hunk 1]
```diff
@@ -30,25 +30,31 @@ def load(self):
...
                 'AWS_SECRET': aws_cli_config.get('default', 'aws_secret_access_key'),
-            }
-            return self._raw_data
-        else:
+            })
+
+        path = getenvpath(self.environment_variable, self.default_path)
+        if os.path.isfile(path):
+            try:
+                raw_data.update(yaml.load(path))
```
Comment (algrs) [inline baiji/config.py]: So if they have different keys, `.bodylabs` is used.

**Comment by algrs** [comment]
👍

File: baiji/config.py (hunk 1 above)
Comment (paulmelnikow) [inline baiji/config.py]: Yes! That seemed like the right way to go. Will add a comment to make that clearer.

**Comment by paulmelnikow** [comment]
Thanks!

**Pull request merged**